pkg/controllers/managementuser/rbac/project_handler.go

```diff
@@ -44,7 +44,8 @@ func (p *pLifecycle) Create(project *v3.Project) (runtime.Object, error) {
 }
 
 func (p *pLifecycle) Updated(project *v3.Project) (runtime.Object, error) {
-	return nil, nil
+	err := p.ensureNamespacesAssigned(project)
```

Comment:
Somehow, this got removed from my forwardport PR, (this is the most important part), so now I will have to forwardport the backport of my forwardport. lol